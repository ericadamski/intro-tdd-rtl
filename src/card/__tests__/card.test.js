import React from "react";
import { render } from "react-testing-library";

import Card from "../index";

describe("Card", () => {
  it("should match the snapshot", () => {
    // Arrange
    const title = "Title";
    const description = "Description";
    const img = "/some/path.png";

    // Act
    const { container } = render(
      <Card title={title} description={description} img={img} />
    );

    // Assert
    expect(container).toMatchSnapshot();
  });
});
