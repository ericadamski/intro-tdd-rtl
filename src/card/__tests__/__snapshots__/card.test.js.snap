// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<div>
  <div
    class="sc-gzVnrw dwXDQI"
  >
    <img
      class="sc-bdVaJa fBmRQa"
      src="/some/path.png"
    />
    <div
      class="sc-bwzfXH iinQpO"
    >
      <div
        class="sc-htpNat gYJYHY"
      >
        <h4
          class="sc-ifAKCX bWdHSd"
        />
        <h2
          class="sc-bxivhb cpAlUE"
        >
          Title
        </h2>
      </div>
      <i
        class="sc-bZQynM eFgNFJ"
      >
        <ion-icon
          name="airplane"
        />
      </i>
      <p
        class="sc-EHOje jvzENE"
      >
        Description
      </p>
    </div>
  </div>
</div>
`;
